compiler/x/codegen/OMRInstOpCode.hpp

```diff
@@ -248,6 +248,7 @@ class InstOpCode: public OMR::InstOpCode
       EVEX_L128 = 0x4,
       EVEX_L256 = 0x5,
       EVEX_L512 = 0x6,
+      VEX_LZ    = 0x8,
```

Comment:
What's the reason this is "8" rather than "7"?  Presumably, increasing this value to now requiring 4 bits is the cause of the `vex_l` field being widened.

Reply:
I need the low bits to be zero so the L bits are 0 in the VEX prefix.

Reply:
OK. Done.